COCO_DATA_FORMAT.md

```diff
@@ -227,4 +227,28 @@ people_dataset/
     train_coco_location.json
     test_coco_location.json
 ```
-```
\ No newline at end of file
+
+
+## KeyValuePair dataset
+
+KeyValuePair dataset is associated with a schema defining a task that input is text + (multi-)image, label is key-value pairs. Each sample of dataset consists of
+- input:
+    - images, each image is optionally associated with text metadata as a dictionary
+    - text (optional), a dictionary with keys being the field name e.g. `{'question': 'a specific question related to the images input'}`
```

Comment:
Your example text input is  "text": "What objects are unique in the first image compared to the second image?", which is not a dictionary below at line 250 and line 251?

Reply:
good catch! fixed